### PR TITLE
doc: prometheus_client cleanup

### PR DESCRIPTION
**Pull request by sobrase**
Deleted unwanted example file for prometheus_client and added it to the readme file.

### CODE REVIEW

File: roles/addons/prometheus_client/readme.rst [hunk 1]
```diff
@@ -11,6 +11,33 @@ Instructions
 
 Node_exporter is available at http://localhost:9100
 
+exporters configuration file to add to inventory/group_vars/equipment_type_X
+depending on where the exporter should be:
```
Comment (btravouillon) [inline roles/addons/prometheus_client/readme.rst:15]: Does the role work out-of-the-box without these parameters?

Comment (oxedions) [reply roles/addons/prometheus_client/readme.rst:15]: Without this, the role will do nothing.

Comment (btravouillon) [reply roles/addons/prometheus_client/readme.rst:15]: Can we consider adding this to defaults/main.yml?

Comment (oxedions) [reply roles/addons/prometheus_client/readme.rst:15]: No, impossible for these ones. These need to be manually tuned for each kind of equipment, no defaults possible here.

Comment (oxedions) [reply roles/addons/prometheus_client/readme.rst:15]: Ho ok got it, you mean an empty configuration. Will do it.

**Comment by oxedions** [comment]
Done in 330.

**Pull request closed without merging**